agent/ecscni/plugin.go

```diff
@@ -210,14 +215,14 @@ type cniPluginVersion struct {
 
 // str generates a string version of the CNI plugin version
 // Example:
-// {"version":"2017.06.0","dirty":true,"gitShortHash":"226db36"} => *226db36V2017.06.0
-// {"version":"2017.06.0","dirty":false,"gitShortHash":"326db36"} => 326db36V2017.06.0
+// {"version":"2017.06.0","dirty":true,"gitShortHash":"226db36"} => @226db36-2017.06.0
+// {"version":"2017.06.0","dirty":false,"gitShortHash":"326db36"} => 326db36-2017.06.0
 func (version *cniPluginVersion) str() string {
 	ver := ""
 	if version.Dirty {
-		ver = "*"
+		ver = "@"
```

Comment:
I have an explanation for this in the commit message: https://github.com/aws/amazon-ecs-agent/pull/879/commits/e7925c706d03973bc069833fba0e356ec8393175 

> ECS Attribute values cannot container asterisks